proyecto/src/main/java/com/proyecto/proyecto/IUsuarios.java

```diff
@@ -9,5 +9,7 @@
     Usuario buscarPorId(Integer idUsuario);
     Usuario buscarPorUsername(String username);
     void setContrasenia(Usuario usuario,String contraseña);
+    int bloquear(int idUsuario);
+	int activar(int idUsuario);
```

Comment:
#should El tema de la indentación de nuevo! Tiene toda la pinta de ser lo de tabulaciones vs espacios